src/target/riscv/riscv-013.c

```diff
@@ -314,7 +314,8 @@ static void decode_dmi(char *text, unsigned address, unsigned data)
 		{ DMI_DMSTATUS, DMI_DMSTATUS_ANYHALTED, "anyhalted" },
 		{ DMI_DMSTATUS, DMI_DMSTATUS_AUTHENTICATED, "authenticated" },
 		{ DMI_DMSTATUS, DMI_DMSTATUS_AUTHBUSY, "authbusy" },
-		{ DMI_DMSTATUS, DMI_DMSTATUS_DEVTREEVALID, "devtreevalid" },
```

Comment:
Did devtreevalid disappear?  If so, is that change directly related to the halt groups?

Reply:
devtreevalid was renamed to confstrptrvalid in the spec. This change is not directly related to halt groups.

This file is auto-generated as part of building the spec, so I just wholesale replaced it with the latest one.

Reply:
Not "this file" but debug_defines.h.